components/shared/Footer.js

```diff
@@ -1,21 +1,76 @@
-import React from "react";
 import Container from "react-bootstrap/Container";
 import Row from "react-bootstrap/Row";
 import Col from "react-bootstrap/Col";
 import Image from "react-bootstrap/Image";
+import { FaRegEnvelope } from "react-icons/fa";
+import IconLabel from "./IconLabel";
 
-export default function Footer(props) {
+export const LINKS = [
+  { label: "About", href: "https://austinmobility.io" },
+  { label: "Data", href: "https://data.austintexas.gov" },
+  {
+    label: "Disclaimer",
+    href: "https://www.austintexas.gov/page/city-austin-open-data-terms-use",
+  },
+  {
+    label: "Code",
+    href: "https://github.com/cityofaustin/atd-data-and-performance",
```

Comment:
and this link can be switched to this project's repo: `https://github.com/cityofaustin/atd-product`

Reply:
oops this was obvious, my bad for not catching it!